// sendbird-uikit/dist/index.js
export { default as SendBirdProvider } from './SendbirdProvider.js';
export { default as App } from './App.js';
export { default as ChannelSettings } from './ChannelSettings.js';
export { default as ChannelList } from './ChannelList.js';
export { default as Channel, getAllEmojisFromEmojiContainer, getEmojiCategoriesFromEmojiContainer, getEmojisFromEmojiContainer } from './Channel.js';
import { S as SendbirdSdkContext } from './LocalizationContext-963dc1a0.js';
export { g as getStringSet, w as withSendBird } from './LocalizationContext-963dc1a0.js';
export { default as OpenChannel } from './OpenChannel.js';
export { default as OpenChannelSettings } from './OpenChannelSettings.js';
export { default as MessageSearch } from './MessageSearch.js';
export { s as sendBirdSelectors } from './index-f66438bd.js';
import { useContext } from 'react';
import 'prop-types';
import 'sendbird';
import './actionTypes-15ff2923.js';
import './index-a1512a62.js';
import 'css-vars-ponyfill';
import './index-6ff1fa1b.js';
import './LeaveChannel-058050f7.js';
import './index-ec4c0e85.js';
import './index-7c2502c4.js';
import './utils-fa79caf2.js';
import './index-7e077832.js';
import './index-a9a603e2.js';
import './index-53ae4315.js';
import './index-1de8106c.js';
import 'react-dom';

/**
 * Example:
 * const MyComponent = () => {
 *  const context = useSendbirdStateContext();
 *  const sdk = sendbirdSelectors.getSdk(context);
 *  return (<div>...</div>);
 * }
 */

function useSendbirdStateContext() {
  var context = useContext(SendbirdSdkContext);
  return context;
}

export { useSendbirdStateContext };
